### PR TITLE
Added support for Updates API endpoint

### PR DESCRIPTION
**Pull request by patpicos**
- Added support for Updates API + relevant struct
- Added 2 tests for new feature

- Updated a few failing tests and disabled some assertion that are "flakey". More work is required on that front

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -148,6 +149,31 @@ func (c *Client) GetSeries(s *Series) error {
 	return nil
 }
 
+// GetUpdates returns a map of show identifiers updated since epoch
+func (c *Client) GetUpdates(epoch int) (Updates, error) {
```
Comment (pioz) [inline client.go]: For consistency with other functions, I will change this function like this:
```
func (c *Client) GetUpdates(epoch int) ([]Updates, error) {
    resp, err := c.performGETRequest("/updated/query", url.Values{"fromTime": {strconv.Itoa(epoch)}})
    if err != nil {
		return nil, err
	}
	defer resp.Body.Close()
	data := new(updatesAPIResponse)
	err = parseResponse(resp.Body, &data)
	if err != nil {
		return nil, err
	}
	return data.Data, nil
}
```

File: updates.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package tvdb
+
+//Updates contains an slice of show ids with their last updated date in Epoch Time
+type Updates struct {
+	UpdateEntry []UpdateEntry `json:"data"`
+}
+
+//UpdateEntry Specifies when a show was last updated
+type UpdateEntry struct {
```
Comment (pioz) [inline updates.go]: Call this type simply `type Update struct`

File: updates.go [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package tvdb
+
+//Updates contains an slice of show ids with their last updated date in Epoch Time
+type Updates struct {
```
Comment (pioz) [inline updates.go]: Move this type in `api_response_types.go`:
```
type updatesAPIResponse struct {
	Data []Update `json:"data"`
}
```

File: client_test.go [hunk 4]
```diff
@@ -36,8 +37,24 @@ func TestClientGetLanguages(t *testing.T) {
...
-	assert.Equal(t, "English", languages[0].EnglishName)
+	assert.True(t, len(languages) > 0, "Ensure more than 0 languages are returned.") //183 supported as of 08/03/2020
+	// assert.Equal(t, "English", languages[0].EnglishName) //disabled. English is not the firt language
+}
+
+func TestClientGetUpdates(t *testing.T) {
+	c := login(t)
+	updates, err := c.GetUpdates(1594509621) //Get all updates
+	assert.Nil(t, err)
+	fmt.Printf("%d needs to be updated\n", len(updates.UpdateEntry))
```
Comment (pioz) [inline client_test.go]: Use `t.Logf` instead

File: client_test.go [hunk 5]
```diff
@@ -36,8 +37,24 @@ func TestClientGetLanguages(t *testing.T) {
...
+	assert.Greater(t, len(updates.UpdateEntry), 0, "There should be many shows to update")
+}
+
+func TestClientGetUpdatesOldDate(t *testing.T) {
+	c := login(t)
+	updates, err := c.GetUpdates(0) //Get all updates
+	assert.Nil(t, err)
+	fmt.Printf("%d needs to be updated\n", len(updates.UpdateEntry))
```
Comment (pioz) [inline client_test.go]: Use `t.Logf` instead

**Review by pioz — CHANGES_REQUESTED** [review]
Thank you very much for your help! It is really appreciated! I've some request changes for you :D

**Comment by patpicos** [comment]
Hi @pioz I have updated the PR by addressing all comments. Good comments :)

**Review by pioz — APPROVED** [review]

**Pull request merged**